pynitrokey/cli/nethsm.py

```diff
@@ -864,16 +890,34 @@ def set_backup_passphrase(
     prompt=True,
     help="The current unlock passphrase",
 )
+@click.option(
+    "-f",
+    "--force",
+    is_flag=True,
+    help="Do not ask for confirmation before changing the passphrase",
+)
 @click.pass_context
 def set_unlock_passphrase(
-    ctx: Context, new_passphrase: str, current_passphrase: str
+    ctx: Context, new_passphrase: str, current_passphrase: str, force: bool
 ) -> None:
     """Set the unlock passphrase of a NetHSM.
 
     Changing the unlock passphrase requires the current passphrase.
 
     This command requires authentication as a user with the Administrator
     role."""
+
+    print(
```

Comment:
In my opinion this warning should only be printed if the force flag is false. But may this is more a matter of taste.